docs/modules/reference/pages/provisioning/adapters/snmp-metadata.adoc

```diff
@@ -32,6 +32,23 @@ The provisioning adapter automatically appends the `.0` value to single-entry OI
 In some edge cases, you will need to specify the entire OID statically to retrieve a single piece of data.
 Specifying the attribute `exact="true"` allows you to specify the entire OID.
 
+There is also an optional attribute `resultsBehavior` that can be used to specify how the adapter should update the node's metadata. The table below outlines the behavior of each option:
+[options="header", cols="1,2"]
+|===
+| Value
+| Description
+
+| `replace`
+| Default behavior. The adapter will replace the node's metadata context `snmp` with the data retrieved from the node.
+
+| `update`
+| The adapter will update the node's metadata context `snmp` with the data retrieved from the node.
+
+ New values will be  added, existing values will be updated, and values not present in the retrieved data will be left unchanged.
+
+ NOTE: This behavior can leave old values in the metadata context if the retrieved data does not contain the same keys.
```

Comment:
Do we have a suggestion for how to deal with this? Is this an issue that might cause a problem?

Reply:
I planned on following this up with a schema change that would add a last changed field next to each Metadata entry regardless of context.

Next I planned on an additional option that would hold a older than in days value.

During each pass from the node it would only clean out entries older than that option's value. 

The default would be null or never clean and only when the update behavior is used.